plone/app/contenttypes/behaviors/collection.py

```diff
@@ -19,14 +19,36 @@
 from zope.schema.vocabulary import SimpleTerm
 from zope.schema.vocabulary import SimpleVocabulary
 
+# Same list that in plone/app/content/browser/contents/__init__.py:get_columns
+COLUMNS_TITLE = {
```

Comment:
Given it is used in the `tabular_fields` property, maybe `TABULAR_FIELD_MAPPING` might be better.

Adding a module variable seems too much to me given we already know we have code duplication with `plone.app.content`.
The code duplication will be hopefully fixed in the future, maybe importingh the same dictionary from a common package.
For this reason I would add the mapping as a`_` prefixed class attribute (e.g. `_tabular_fields_mapping`).

That would allow us to deal with the code duplication with `plone.app.content` in the future with more freedom and fear to break things.

Reply:
Yeah it's sad for the code duplication. I moved the vocabulary to a dedicated vocabulary.py module, still inside plone.app.contenttypes though for better clarity.
In https://github.com/plone/plone.app.content/blob/b7406aa1389e83f26b3ee4ed1282912a984fa32f/plone/app/content/browser/contents/__init__.py#L205-L222 all labels are translated to return a json with translated label. This can maybe be modified there to consume the vocabulary, iterate over the terms and translate them.

Reply:
There shouldn't be tabular in the name I think. Initially I used "column" because of brain column metadata, not table column. Anyway I renamed it to simple field_label.

Reply:
For code deduplication, see https://github.com/plone/plone.app.content/pull/203